mwm-5-dynamic-delivery/presentation/PITCHME.md

```diff
@@ -306,6 +306,13 @@ if (!manager.installedModules.contains(name)) {
 
 ---
 
+### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">User consent</span><span style="text-transform: none; font-size:0.8em;"> to download the module</span>
+
+- If a module, or a list of modules need to be downloaded, during a certain period of time, are heavier than 10MB, the user's consent must be requested.
```

Comment:
Maybe `will be` instead of `must be` as the Google lib does it automatically.

Reply:
That's false (https://developer.android.com/guide/app-bundle/playcore#obtain_confirmation)
We have to add some code to display the consent dialog.
Otherwise, nothing will be displayed and downloaded.

Reply:
👏